.github/workflows/tag-maintainers.yml

```diff
@@ -0,0 +1,235 @@
+name: Sync Plugin Maintainer Reviewers
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review, reopened, synchronize]
+
+# Concurrency settings to ensure that only one instance of this workflow runs per PR.
+# If a new commit is pushed, it cancels the previous run.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read       # To checkout code
+  pull-requests: write # To add/remove reviewers and comment
+
+jobs:
+  tag-maintainers:
+    runs-on: ubuntu-latest
+    if: |
+      github.repository_owner == 'nix-community' &&
```

Comment:
If we have this check in the condition, there is no reason to have `permissions` above.

`permissions: {}` will explicitly set no permissions for `github.token`.

If the intention of the permissions block was to allow testing on forks that don't have an App installed, then we need to drop the `owner` condition.

Reply:
Yeah, that is half baked.. I don't really feel like this benefits from fork support the same as some other flows.